README.md

```diff
@@ -1,50 +1,94 @@
-```
+Note that the external-provisioner does not scale with more replicas. Only one external-provisioner is elected as leader and running. The others are waiting for the leader to die. They re-elect a new active leader in ~15 seconds after death of the old leader.
 
-### Delete PVC
+### Command line options
 
-```bash
-kubectl delete pvc example-pvc
-```
+#### Important optional arguments that are highly recommended to be used
+* `--csi-address <path to CSI socket>`: This is the path to the CSI driver socket inside the pod that the external-provisioner container will use to issue CSI operations (`/run/csi/socket` is used by default).
+
+* `--enable-leader-election`: Enables leader election. This is useful when there are multiple replicas of the same external-provisioner running for one CSI driver. Only one of them may be active (=leader). A new leader will be re-elected when current leader dies or becomes unresponsive for ~15 seconds.
+
+* `--timeout <duration>`: Timeout of all calls to CSI driver. It should be set to value that accommodates majority of `ControllerCreateVolume` and `ControlerDeleteVolume` calls. See [CSI error and timeout handling](#csi-error-and-timeout-handling) for details. 15 seconds is used by default.
+
+* `--retry-interval-start <duration>` - Initial retry interval of failed provisioning or deletion. It doubles with each failure, up to `--retry-interval-max` and then it stops increasing. Default value is 1 second. [CSI error and timeout handling](#csi-error-and-timeout-handling) for details.
+
+* `--retry-interval-max <duration>` - Maximum retry interval of failed provisioning or deletion. See [CSI error and timeout handling](#csi-error-and-timeout-handling) for details.
+
+* `--worker-threads <num>`: Number of simultaneously running `ControllerCreateVolume` and `ControllerDeleteVolume` operations. Default value is `100`.
+
+#### Other recognized arguments
+* `--provisioning-retry-count <num>`: Sets number of retries of failed provisioning of a single volume. The external-provisioner will give up provisioning after given number of retries. It should be used only with special CSI drivers whose provisioning of volumes is extremely slow (e.g. hours or days) or expensive so the external-provisioner does not waste resources. Set to `0` to retry forever (which is also the default value).
+
+* `--deletion-retry-count <num>`: Sets number of retries of failed deletion of a single volume. The external-provisioner will give up deletion of the volume after given number of retries. It should be used only with special CSI drivers whose deletion of volumes is extremely slow (e.g. hours or days) or expensive so the external-provisioner does not waste resources. Set to `0` to retry forever (which is also the default value).
+
+* `--feature-gates <gates>`: Enables or disables alpha (disabled by default) or beta features (enabled by default). See [list of features](#feature-status) or `--help` output for list of recognized features.
```

Comment:
i'm thinking topology will not be enabled by default even though it's beta

Reply:
Also, example syntax?

Reply:
Fixed